src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -54,6 +52,9 @@
 import javax.swing.undo.CannotRedoException;
 import javax.swing.undo.CannotUndoException;
 
+import ca.odell.glazedlists.event.ListEventListener;
+import com.jgoodies.forms.builder.FormBuilder;
+import com.jgoodies.forms.layout.FormLayout;
 import net.sf.jabref.BibDatabaseContext;
```

Comment:
According to https://github.com/JabRef/jabref/issues/1091, there should be an empty line before the JabRef-specific imports.